binding.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/flamego/flamego"
 	"github.com/flamego/validator"
+	"gopkg.in/yaml.v3"
```

Comment:
```suggestion

	"gopkg.in/yaml.v3"

	"github.com/flamego/flamego"
	"github.com/flamego/validator"
```

ref: https://github.com/unknwon/go-code-convention/blob/main/en-US.md#import-packages

Reply:
@Germiniku This has not been addressed.